modules/access-control/package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "@wso2is/access-control",
+    "version": "1.0.0",
```

Comment:
This has to be same as version in `lerna.json`. FYI
```suggestion
    "version": "^1.2.227",
```

Reply:
Fixed with commit 06fcf6e